.cirrus.yml

```diff
@@ -272,15 +280,15 @@ sanitizer_task:
   container:
     # Just uses a recent/common distro to run memory error/leak checks.
     dockerfile: ci/ubuntu-20.04/Dockerfile
-    cpu: 4
+    cpu: 3
     # AddressSanitizer uses a lot more memory than a typical config.
-    memory: 12GB
+    memory: 16GB
   << : *CI_TEMPLATE
   test_fuzzers_script: ./ci/test-fuzzers.sh
   coverage_script: ./ci/upload-coverage.sh
   env:
     CXXFLAGS: -DZEEK_DICT_DEBUG
-    ZEEK_CI_CONFIGURE_FLAGS: *SANITIZER_CONFIG
+    ZEEK_CI_CONFIGURE_FLAGS: *SANITIZER_CONFIG *WITHOUT_SPICY_CONFIG
```

Comment:
Currently `ZEEK_INCLUDE_PLUGINS` does not seem to correctly set up link dependencies for `zeek_fuzzer_shared` and with Spicy enabled we'd see this error:

```
/usr/bin/ld: libzeek_fuzzer_shared.so: undefined reference to `plugin::_Zeek_Spicy::__bif_functions_init(zeek::plugin::Plugin*)'
/usr/bin/ld: libzeek_fuzzer_shared.so: undefined reference to `plugin::_Zeek_Spicy::__bif_consts_init(zeek::plugin::Plugin*)'
/usr/bin/ld: libzeek_fuzzer_shared.so: undefined reference to `plugin::_Zeek_Spicy::__bif_events_init(zeek::plugin::Plugin*)'
```